doc/book/admin/upgrades/1.6-1.10.rst

```diff
@@ -0,0 +1,59 @@
+..  _admin-upgrades-1.6-1.10:
+
+Live upgrade from Tarantool 1.6 to 1.10
+=======================================
+
+This page includes explanations and solutions to some common issues
+when upgrading a replica set from Tarantool 1.6 to 1.10.
+
+..  include:: ../_includes/1.6-to-2.x-condition.rst
+
+Let's first reiterate the upgrade procedure for any Tarantool version:
```

Comment:
May be to rephrase this intro sentence somehow? 
The previous sentence says, "However, a direct upgrade of a replica set from 1.6 to 2.x is also possible, but only with downtime."
And then we have this intro and the procedure description, but it's not clear that this is the generic procedure regardless the TT version.